### PR TITLE
DICT-445 leeuwenhoek treatment updates

### PR DESCRIPTION
**Pull request by bullenc**
https://gdc-ctds.atlassian.net/browse/DICT-445

### CODE REVIEW

**Review by stilesj-uchicago — APPROVED** [review]
lgtm

**Review by wwysoc2 — APPROVED** [review]
👍

File: src/gdcdictionary/schemas/treatment.yaml [hunk 1]
```diff
@@ -172,6 +176,37 @@ properties:
...
+      - mg/m2/wk
+      - mg/mL
+      - mg/wk
+      - mIU
+      - mL
+      - ug
+      - ug/m2
+      - Wafer
+      - Unknown
+      - Not Reported
```
Comment (wteouchicago) [inline src/gdcdictionary/schemas/treatment.yaml:208]: should we update the DICT ticket to indicate that these two are to be added?

Comment (bullenc) [reply src/gdcdictionary/schemas/treatment.yaml:208]: Yes, thank you

**Review by wteouchicago — APPROVED** [review]

**Pull request merged**